spec/journal_title_spec.rb

```diff
@@ -291,7 +291,7 @@
   end # the state
 
   context "the world" do
-    it_behaves_like "great results for format journal", "The world" do
+    it_behaves_like "great results for format journal", "The world", :fixme => true do
```

Comment:
I no longer remember - did we give up on "The world" journal?  Or did you just forget to remove the fixme?

Reply:
I kept the fixme because the linked 880's containing the parallel titles are legitimate in cataloging and affecting the results. I'm not sure there is a way to fix the test (or how we index the records).

Reply:
oh yeah!  Now I remember.

Reply:
should this be in the comments somewhere?